package/CHANGELOG

```diff
@@ -13,28 +13,30 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/16 kain88-de, jbarnoud
+??/??/16 tyler.je.reddy, kain88-de, jbarnoud, richardjgowers
 
-  * 0.13.1
+  * 0.14.0
```

Comment:
These changes can stay in a 0.13.1 because they don't break the API in any way. Ultimately, @richardjgowers is responsible for assigning these numbers. If Richard wants this to be 0.14.0 then leave it, otherwise change back.